src/overlays/actors/ovl_En_Bjt/z_en_bjt.c

```diff
@@ -31,49 +58,408 @@ const ActorInit En_Bjt_InitVars = {
+    }
+}
+
+s32 EnBjt_TakeItem(s32 exchangeItem) {
+    switch (exchangeItem) {
+        case EXCH_ITEM_LETTER_TO_KAFEI:
+            func_801149A0(ITEM_LETTER_TO_KAFEI, SLOT(ITEM_LETTER_TO_KAFEI));
+            break;
+
+        case EXCH_ITEM_DEED_SWAMP:
+            func_801149A0(ITEM_DEED_SWAMP, SLOT(ITEM_DEED_SWAMP));
+            break;
+
+        case EXCH_ITEM_DEED_MOUNTAIN:
+            func_801149A0(ITEM_DEED_MOUNTAIN, SLOT(ITEM_DEED_MOUNTAIN));
+            break;
+
+        case EXCH_ITEM_DEED_OCEAN:
+            func_801149A0(ITEM_DEED_OCEAN, SLOT(ITEM_DEED_OCEAN));
+            break;
+
+        case EXCH_ITEM_DEED_LAND:
+            func_801149A0(ITEM_DEED_LAND, SLOT(ITEM_DEED_LAND));
+            break;
+
+        case EXCH_ITEM_LETTER_MAMA:
+            func_801149A0(ITEM_LETTER_MAMA, SLOT(ITEM_LETTER_MAMA));
+            break;
+
+        default:
+            break;
+    }
+    return 0;
+}
+
+#define FULLY_GROWN_SCALE 0.017f
+
+/**
+ * @return boolean true if scale is set to final value
+ */
+s32 EnBjt_Appear(EnBjt* this) {
+    s32 finished = false;
+
+    if (!this->playedSfx) {
+        Actor_PlaySfxAtPos(&this->actor, NA_SE_EV_TOILET_HAND_APPEAR);
+        this->playedSfx = true;
+    }
+
+    Math_ApproachF(&this->actor.scale.x, FULLY_GROWN_SCALE, 0.21f, 0.3f);
+    if ((FULLY_GROWN_SCALE - this->actor.scale.x) < FULLY_GROWN_SCALE / 100.0f) {
+        this->actor.scale.x = FULLY_GROWN_SCALE;
+        this->stateFlags |= TOILET_HAND_STATE_VISIBLE;
+        this->stateFlags &= ~TOILET_HAND_STATE_APPEARING;
+        finished = true;
+    }
+    this->heightOffset = (this->actor.scale.x / FULLY_GROWN_SCALE) * 4.0f;
+    this->actor.world.pos.y = this->actor.home.pos.y + this->heightOffset;
+    Actor_SetScale(&this->actor, this->actor.scale.x);
+
+    return finished;
+}
+
+/**
+ * @return boolean true if scale is set to final value
+ */
+s32 EnBjt_Vanish(EnBjt* this) {
+    s32 finished = false;
+
+    if (!this->playedSfx) {
+        Actor_PlaySfxAtPos(&this->actor, NA_SE_EV_TOILET_HAND_VANISH);
+        this->playedSfx = true;
+    }
+
+    Math_ApproachF(&this->actor.scale.x, 0.0f, 0.21f, 0.3f);
+    if (this->actor.scale.x < FULLY_GROWN_SCALE / 100.0f) {
+        this->actor.scale.x = 0.0f;
+        this->stateFlags &= ~TOILET_HAND_STATE_VANISHING;
+        finished = true;
+    }
+    this->heightOffset = (this->actor.scale.x / FULLY_GROWN_SCALE) * 4.0f;
+    this->actor.world.pos.y = this->actor.home.pos.y + this->heightOffset;
+    Actor_SetScale(&this->actor, this->actor.scale.x);
+
+    return finished;
+}
+
+typedef enum {
+    /* 0 */ TOILET_HAND_BEHAVIOUR_WAIT_FOR_ITEM, // and check if item is suitable
+    /* 1 */ TOILET_HAND_BEHAVIOUR_TAKE_ITEM,     // take paper and disappear to use it
+    /* 2 */ TOILET_HAND_BEHAVIOUR_USE_ITEM,      // using paper, flushing
+    /* 3 */ TOILET_HAND_BEHAVIOUR_REAPPEAR,      // reappear with reward
+    /* 4 */ TOILET_HAND_BEHAVIOUR_REWARD_GIVEN,
+} ToiletHandBehaviour;
```

Comment:
`Behaviour` vs. `Behavior`, both are correct depending on the region, and both are in the repo, although `Behavior` seems much more common. Not sure if we should nitpick the `o` vs `ou` spelling

Reply:
Let's just sort it out later, wouldn't be our only inconsistency.

Reply:
I agree with Elliptic that this specifically is better decided later.